core/src/main/scala/org/apache/spark/deploy/history/HistoryServer.scala

```diff
@@ -52,7 +52,8 @@ class HistoryServer(
     provider: ApplicationHistoryProvider,
     securityManager: SecurityManager,
     port: Int)
-  extends WebUI(securityManager, securityManager.getSSLOptions("historyServer"), port, conf)
+  extends WebUI(securityManager, securityManager.getSSLOptions("historyServer"),
+    port, conf, name = "HistoryServerUI")
```

Comment:
instead of the `name` parameter, how about `poolSize` parameter directly?

Reply:
ok